pkg/utils/test/setup_helpers.go

```diff
@@ -183,6 +189,27 @@ func BuildMTLSCredentials(ctx context.Context, k8sClient *kubernetes.Clientset,
 	}
 }
 
+// ExtractModuleVersion extracts version of an imported module in go.mod.
+// If the module is not found, or we failed to parse the module version, it will return an error.
+func ExtractModuleVersion(moduleName string) (string, error) {
+	// TODO: use a path non relevant to the file itself
```

Comment:
How about adding a test util package that will provide access to the project's root absolute path (like here: https://stackoverflow.com/a/38644571) already in this PR? I'd suggest putting a `ProjectRootPath()` function in `test/helpers/projectroot.go`.

Reply:
Added a `ProjectRootPath` function.